components/event-header.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="event-header">
+    <h1 class="event-header__name">
+      <span class="event-header__location">{{ location }}</span>
+      {{ name }}
+    </h1>
+    <nav class="language-switch">
+      <ul class="language-switch__list">
+        <li v-for="{ locale } in _allNameLocales" :key="locale">
+          <nuxt-link
+            class="language-switch__link"
+            :class="{ 'language-switch__link--active': route.params.language === locale }"
+            rel="alternate"
+            :hreflang="locale"
+            :lang="locale"
+            :to="{ name: route.name, params: { ...route.params, language: locale } }"
+          >
+            {{ getNativeName(locale) }}
+          </nuxt-link>
+        </li>
+      </ul>
+    </nav>
+  </div>
+</template>
+
+<script>
+  import ISO6391 from 'iso-639-1';
+
+  export default {
+    methods: {
+      getNativeName: ISO6391.getNativeName,
+    },
+    props: {
+      name: String,
+      location: String,
+      _allNameLocales: Array,
```

Comment:
Maybe rename this to something like `locales`?

Reply:
Since the props are destructed from event will have to rename it in the GraphQL query, which is possible but I don't like to do since it obscures where data comes from.